source/agora/consensus/Validation.d

```diff
@@ -990,6 +993,62 @@ public string isInvalidReason (const ref Block block, in ulong prev_height,
     return null;
 }
 
+/*******************************************************************************
+
+    Check the validator of an enrollment.
+
+    A Validator's enrollment is considered valid if:
+        - UTXO is unspent frozen utxo
+        - Signatures are authentic
+        - The frozen amount must be equal to or greater than 40,000 BOA
+        - The block height shall be greater than the unlock_height of the UTXO.
+
+    Params:
+        block_height = the height of the direct ancestor of this block
+        enrollment = The enrollment of the target to be verified
+        findUTXO = delegate to find the referenced unspent UTXOs with
+
+    Returns:
+        `null` if the validator's UTXO is valid, a string explaining the reason it
```

Comment:
`a string explaining the reason it is invalid otherwise.` => `otherwise a string explaining the reason it is invalid`

Reply:
applied the suggest.